spec/assigner_spec.rb

```diff
@@ -4,15 +4,17 @@
   context 'named special cases' do
     context 'weekly meetings' do
       it 'should assign the meetings' do
-        a = Assigner.new '92345678: internal weekly meeting'
+        a = Assigner.new '92345678: internal weekly meeting : : 2017-04-25T09:02:30+00: 00'
```

Comment:
@pascalandermatt if you select a tag int toggle, `: :` will not be in the string. e.g.

`571115323: bla bla bla : Development : 2017-04-26T08:05:00+00: 00`